Cesium3DTilesReader/test/TestTilesetReader.cpp

```diff
@@ -15,6 +13,24 @@
 #include <string>
 #include <vector>
 
+using namespace doctest;
+
+bool compareVectors(
```

Comment:
This should be in the anonymous namespace. Or maybe moved to `CesiumNativeTests`?